modules/terraform/validate.go

```diff
@@ -12,9 +12,15 @@ func Validate(t testing.TestingT, options *Options) string {
 	return out
 }
 
-// ValidateE calls terraform validate and returns stdout/stderr.
+// ValidateE calls terraform validate and returns stdout/stderr. Terragrunt's binary expects the `validate-inputs` command, whereas Terraform's correlative command is `validate`
```

Comment:
Ahhhh `validate-inputs` and `validate` are two separate commands, so we should probably have a `ValidateInputsE` function that calls `validate-inputs` and asserts the binary is terragrunt (similar to `TgApplyAllE`).

Reply:
Done in 2b755e7